sql/udfs/use-udfs-in-javascript.mdx

```diff
@@ -37,14 +39,53 @@ See the correspondence between SQL types and JavaScript types in the [Data type
 
 If the function you defined returns a table, you need to use the `yield` statement to return the data of each row. For example:
 
-```js
+```sql
 CREATE FUNCTION series(n int) RETURNS TABLE (x int) LANGUAGE javascript AS $$
     for(let i = 0; i < n; i++) {
         yield i;
     }
 $$;
 ```
 
+### Async and batched JavaScript UDFs
+
+<Note>
+Added in v2.3.
+</Note>
+
+You can create JavaScript UDFs with asynchronous execution and batch processing using the following syntax.
+
+```sql Syntax
+CREATE FUNCTION function_name(arg_name arg_type [, ...])
+    RETURNS return_type
+    LANGUAGE javascript
+    AS $$
+export async function function_name(arg_name) {
+    ... // You can call async functions such as `await fetch()` here.
+}
+$$ WITH (
+    async = [true|false],
+    batch = [true|false]
+);
+```
+
+- `async`: Set to `true` to enable async functions like `await fetch()`.
+- `batch`: Set to `true` to process multiple batches.
```

Comment:
May need to add something like: When this is `true`, each argument of the function should accept an array of `arg_type`.